apis/slo/v1alpha1/nodemetric_types.go

```diff
@@ -49,6 +49,8 @@ type PodMetricInfo struct {
 	Name      string      `json:"name,omitempty"`
 	Namespace string      `json:"namespace,omitempty"`
 	PodUsage  ResourceMap `json:"podUsage,omitempty"`
+	// Third party extensions for PodMetric
+	Extensions *MetricExtensions `json:"extensions,omitempty"`
```

Comment:
consider use `ExtensionsMap` in nodeslo_ext.go？

Reply:
SGTM